README.md

```diff
@@ -56,6 +56,8 @@ which is used for local run/debug, because the integration tests will remove all
             - `amountPlanned`
             - `cancelUrl` (custom field)
             - `successUrl` (custom field)
+            - `reference` (custom field, up to 116 characters)
+            - `languageCode` (custom field)
```

Comment:
Why is "languageCode" it mandatory now?

Reply:
actually, it always was mandatory, but never documented though:
https://github.com/commercetools/commercetools-paypal-plus-integration/blob/57d46adfe6e4b0e8b625498fd37eaa9d340e5db2/src/main/resources/ctp/types/payment-paypal.json#L10-L20